### PR TITLE
Logic for non geopath member tiles

### PR DESCRIPTION
**Pull request by jhurlb**
# GOAL
Have routes for non geopath member tiles

### CODE REVIEW

File: tilestache.cfg.sample [hunk 1]
```diff
@@ -38,6 +38,38 @@
                 "name": "mbtiles",
                 "tileset": "data/mbtiles/markernoven.mbtiles"
             }
+        },
+        "productsnogeopath":
+        {
+            "provider":
+            {
+                "name": "mbtiles",
+                "tileset": "data/products-nogeopath.mbtiles"
```
Comment (losinggeneration) [inline tilestache.cfg.sample]: Would we want to, by default, put these in a subdirectory similar to the ones above defaulting to `data/mbtiles`?

**Review by losinggeneration — APPROVED** [review]
Looks like fine. One pretty minor/optional question about organization.

File: tilestache.cfg.sample (hunk 1 above)
Comment (jhurlb) [reply tilestache.cfg.sample]: Nice catch. These should have the same structure.

**Review by losinggeneration — APPROVED** [review]

**Comment by losinggeneration** [comment]
I'm going to merge this. Hopefully the deployment doesn't just take the sample and use that (or if it does, that it doesn't fail if the tiles don't exist in that directory.)

**Comment by jhurlb** [comment]
> I'm going to merge this. Hopefully the deployment doesn't just take the sample and use that (or if it does, that it doesn't fail if the tiles don't exist in that directory.)

Hahah, no it doesn't do that.

**Pull request merged**